deployment/charts/catalog-enricher-service/values.yaml

```diff
@@ -3,7 +3,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 qe-quarkus-app:
-  replicaCount: 1
+  autoscaler:
+    minReplicas: 1
+    maxReplicas: 1
```

Comment:
Why not up to 3?

Reply:
Because current Kafka topic has a partition = 1